packages/web-app-files/src/services/folder/spaces/loaderPersonal.ts

```diff
@@ -37,15 +37,18 @@ export class FolderLoaderSpacesPersonal implements FolderLoader {
         )
         resources = resources.map(buildResource)
 
-        const currentFolder = resources.shift()
-        const hasShareJail = useCapabilityShareJailEnabled(store)
-        yield store.dispatch('Files/loadSharesTree', {
-          client: clientService.owncloudSdk,
-          path: currentFolder.path
-        })
+        const currentFolder =
+          resources.length > 1 || resources[0].isFolder ? resources.shift() : undefined
```

Comment:
Can you add braces here (and the legacy copy of the code)? 
Had to look multiple times what actually was going on with the or and ternary operators...